src/components/map/Map.tsx

```diff
@@ -135,6 +135,12 @@ export default function Map() {
                   <div className="popup-text">
                     { getGeography(record.city, record.state, record.country) }
                   </div>
+                  <div className="popup-heading">
```

Comment:
We should render this conditionally (only if record.sampling_start_date & record.sampling_end_date are not NULL)